PyMPDATA/options.py

```diff
@@ -142,7 +145,11 @@ def __eq__(self, other):
 
     @property
     def n_halo(self) -> int:
-        """halo extent for a given options set"""
+        """Halo extent for a given options set.
+        The halo extent is the number of 'ghost' cells that need to be added
```

Comment:
"number of `ghost` cells" might be misleading as it is rather the number of "ghost layers" in each dimensions, and it is treated differently for vector and scalar fields. E.g., `n_halo=1` might actually mean millions of ghost cells.

Reply:
Changed the phrasing to ghost layers